### PR TITLE
fix: Modal styles and data loading

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR does the following:
- Adds a bar to split the modal text from the list.
- Changes the styles of the Profile component so the user's address / text is bigger and is centered.
- Fixes how picks were fetched.

Screenshot 2023-04-12 at 16 50 37
Screenshot 2023-04-12 at 16 50 24

### CODE REVIEW

File: webapp/src/modules/vendor/decentraland/favorites/api.ts [hunk 1]
```diff
@@ -12,6 +12,25 @@ export const MARKETPLACE_FAVORITES_SERVER_URL = config.get(
   'MARKETPLACE_FAVORITES_SERVER_URL'
 )!
 
+type PaginatedResponse<T> = {
+  results: T[]
+  total: number
+  page: number
+  pages: number
+  limit: number
+}
```
Comment (kevinszuchet) [inline webapp/src/modules/vendor/decentraland/favorites/api.ts:21]: Definitely, we need this type in a common place (@dcl/schemas).

File: webapp/src/components/Modals/FavoritesModal/FavoritesModal.tsx [hunk 2]
```diff
@@ -68,6 +68,7 @@ const FavoritesModal = ({ metadata: { itemId }, onClose }: Props) => {
       <div style={style} tabIndex={0}>
         {isItemLoaded(index) ? (
           <LinkedProfile
+            className={styles.user}
```
Comment (kevinszuchet) [inline webapp/src/components/Modals/FavoritesModal/FavoritesModal.tsx:71]: Where does this style come from? I can't see it in the `FavoritesModal.module.css`.

File: webapp/src/modules/vendor/decentraland/favorites/api.ts (hunk 1 above)
Comment (LautaroPetaccio) [reply webapp/src/modules/vendor/decentraland/favorites/api.ts:21]: We definitely do

File: webapp/src/components/Modals/FavoritesModal/FavoritesModal.tsx (hunk 2 above)
Comment (LautaroPetaccio) [reply webapp/src/components/Modals/FavoritesModal/FavoritesModal.tsx]: I completely forgot to remove it, great catch!

**Review by kevinszuchet — APPROVED** [review]

**Pull request merged**